api/features/views.py

```diff
@@ -103,9 +104,9 @@ class FeatureViewSet(viewsets.ModelViewSet):
 
     def get_serializer_class(self):
         return {
-            "list": ListCreateFeatureSerializer,
-            "retrieve": ListCreateFeatureSerializer,
-            "create": ListCreateFeatureSerializer,
+            "list": FeatureSerializerWithMetadata,
```

Comment:
Do we need the metadata on the list endpoint?

Reply:
Understanding how it was working in the environments, yes.